src/gromit-mpx.c

```diff
@@ -41,7 +41,8 @@ GromitPaintContext *paint_context_new (GromitData *data,
 				       GdkRGBA *paint_color, 
 				       guint width,
 				       guint arrowsize,
-				       guint minwidth)
+				       guint minwidth,
+				       guint maxsize)
```

Comment:
Shouldn't this be called maxwidth to be in line with minwidth?

Reply:
I thought it would be clearer, since maxwidth already exists in data, and the issue specifically mentioned maxsize. Nevertheless, I changed it to maxwidth as you requested.